aip/general/0203.md

```diff
@@ -204,6 +224,8 @@ surpass the costs to clients and API users of not doing so.
 
 ## Changelog
 
+- **2023-07-21**: Clarify that nested behavior and parent behavior are
```

Comment:
We should fix the date before submitting...

Reply:
Fixed!